internal/util/paramtable/service_param.go

```diff
@@ -518,10 +520,24 @@ func (p *MinioConfig) initRootPath() {
 
 func (p *MinioConfig) initUseIAM() {
 	useIAM := p.Base.LoadWithDefault("minio.useIAM", DefaultMinioUseIAM)
-	p.UseIAM, _ = strconv.ParseBool(useIAM)
+	var err error
+	p.UseIAM, err = strconv.ParseBool(useIAM)
+	if err != nil {
+		panic("parse bool useIAM:" + err.Error())
+	}
+}
+
+func (p *MinioConfig) initCloudProvider() {
+	p.CloudProvider = p.Base.LoadWithDefault("minio.cloudProvider", DefaultMinioCloudProvider)
+	supportedCloudProvider := map[string]bool{
```

Comment:
supported cloudProvider should be declared as a constant, both chunk manager initialization and here can use it.

Reply:
updated